test/specs/scale.logarithmic.tests.js

```diff
@@ -1039,47 +1039,40 @@ describe('Logarithmic Scale tests', function() {
 			{
 				dataset: [],
 				scale: {min: 0},
-				firstTick: 1, // value of the first tick
 				lastTick: 10, // value of the last tick
 				describe: 'empty dataset, min: 0, without max'
 			},
 			{
 				dataset: [],
 				scale: {min: 0, max: 80},
-				firstTick: 1,
 				lastTick: 80,
 				describe: 'empty dataset, min: 0, max: 80'
 			},
 			{
 				dataset: [],
 				scale: {min: 0, max: 0.8},
-				firstTick: 0.1,
 				lastTick: 0.8,
 				describe: 'empty dataset, min: 0, max: 0.8'
 			},
 			{
 				dataset: [{x: 0, y: 0}, {x: 10, y: 10}, {x: 1.2, y: 1.2}, {x: 25, y: 25}, {x: 78, y: 78}],
-				firstTick: 1,
 				lastTick: 80,
 				describe: 'dataset min point {x: 0, y: 0}, max point {x:78, y:78}, minNotZero {x: 1.2, y: 1.2}'
```

Comment:
Should probably remove references to `minNotZero` in the tests if we're removing it

Reply:
Done